### PR TITLE
[dv/xcelium] 1 attempt of cleaning up the coverage files

### PR DESCRIPTION
**Pull request by rasmus-madsen**
1 attempt of cleaning up the ccf files for xcelium coverage.
there was a few doublicates and some rules that would overwrite others

I suspect a bit more cleaning up is needed.

@weicaiyang could you run this on your end make sure it collects what we need and nothing we don't

I will look at it some more next week but I would appreciate some feedback

### CODE REVIEW

File: hw/dv/tools/xcelium/common.ccf [hunk 1]
```diff
@@ -48,7 +48,8 @@ set_fsm_reset_scoring
 select_functional  -ams_control  -imm_asrt_class_package
 
 // Improve the scoping and naming of covergroup instances.
-set_covergroup -new_instance_reporting
+// including this added scoring of cip group and others as individual groups
+//set_covergroup -new_instance_reporting
```
Comment (weicaiyang) [inline hw/dv/tools/xcelium/common.ccf:52]: from the comment, looks like you wanted to enable this, but you actually commented this out

**Review by weicaiyang — APPROVED** [review]
With some helps from @cindychip, we see the DV code coverage is gone. This looks great.

Left a question, otherwise, LGTM

File: hw/dv/tools/xcelium/common.ccf (hunk 1 above)
Comment (rasmus-madsen) [reply hw/dv/tools/xcelium/common.ccf:52]: @weicaiyang no I just forgot to delete this
I have cleaned it up.

**Comment by weicaiyang** [comment]
private CI times out. That's weird. This update won't be used in private CI. Can you rebase to the latest and push again?

**Pull request merged**